webpack.config.js

```diff
@@ -0,0 +1,76 @@
+const devCerts = require("office-addin-dev-certs");
+const { CleanWebpackPlugin } = require("clean-webpack-plugin");
+const CopyWebpackPlugin = require("copy-webpack-plugin");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const webpack = require("webpack");
+
+module.exports = async (env, options) => {
+    const dev = options.mode === "development";
+    const config = {
+        devtool: "source-map",
+        entry: {
```

Comment:
Is there a reason why we are creating separate bundles for fallbackauthtaskpane and taskpane? If they are both going to be loaded synchronously from a file called taskpane.html, then it wouldn't seem like they need to be distinct bundles.  UPDATE: Ok. I see now that fallbackauthtaskpane.ts is just misnamed. It's actually going to run in the dialog, so it makes sense that it's a separate bundle. Please rename the bundle to fallbackauth_dialog_. And rename the file to fallbackauth_dialog_.ts.